### ISSUE TITLE
Publish as a pre-built extension in a python with support for Jupyterlab 4.x as well

### ISSUE THREAD

**Issue by mcrutch** [quigleyj97/jupyterlab-spreadsheet#60]
JupyterLab moved to wanting extensions installed as python packages in pre-built typescript so jupyterlab builds aren't required after install and packages can be installed from a single location.

There are lots of examples of how this is done in the jupyterlab extension examples.

Any chance of jupyterlab-spreadsheet getting updated for this?

**Comment by quigleyj97** [comment]
Yes, I'm looking into this to support it for the JupyterLab 4.0 release
